content/plugins/_index.en.md

```diff
@@ -3,10 +3,79 @@ title: Plugins
 weight: 90
 ---
 
+## What is a plugin
 Notepad++ is very extensible using so called plugins. Plugins are small or big additions to Notepad++ to enhance its functionality. Notepad++ comes bundled with a few plugins (when using the installer, you can choose which ones to add), but you can always add your own or remove some. The plugins are located in the Plugins directory in the main Notepad++ installation directory. They are DLL files and simply removing or adding them is enough.
 
-When Notepad++ starts, it looks into its Plugins configuration folder and loads whatever dll it finds. Later, you can add plugins using the Settings -> Import -> Import plugins menu. Be careful to make sure your version of Notepad++ is up-to-date enough for new plugins and that the plugin is compatible with Notepad++ (some very old plugins will not work with newer versions of Notepad++, it is up to the developer of the plugin to add support or not).
 
-Currently, Notepad++ can be found in an (older) ANSI version and the newer Unicode version. The plugin has to match this version, otherwise it will not work (Notepad++ will warn you about this). Plugins can be found anywhere on the internet, but a large can be found in the Notepad++ Plugins project on SourceForge.net: http://sourceforge.net/projects/npp-plugins/. There is even a plugin, called Plugin Manager, which works out the installation details and version management for you.
+## How to install a plugin
+
+### Install using Plugins Admin
+The Plugins Admin allows you to easily install plugins that are in the Plugins List.  To do so, place a check mark next to the Plugin(s) you wish to install, then select Install.
+
+### Install plugin manually
+If the plugin you want to install is not listed in the Plugins Admin, you may still install it manually.  The plugin (in the DLL form) should be placed in the \plugins subfolder of the Notepad++ Install Folder, under the subfolder with the same name of plugin binary name without file extension.
+For example, if the plugin you want to install named `myAwesomePlugin.dll`, you should install it with the following path:
+`%PROGRAMFILES(x86)%\Notepad++\plugins\myAwesomePlugin\myAwesomePlugin.dll`
+
+Once you installed the plugin, you can use (and you may configure) it via the menu "Plugins".
+
+## How to develop a plugin
+
+### Getting started - Make your plugin in less 10 minutes (C++)
+
+Here are the instructions to make your first Notepad++ plugin in less 10 minutes, by following 6 steps:
+
+1. Download and unzip [Notepad++ Plugin Template](https://github.com/npp-plugins/plugintemplate/archive/v3.zip).
```

Comment:
Shouldn't it be https://github.com/npp-plugins/plugintemplate/releases/download/v3.1/pluginTemplate.v3.1.bin.zip and https://github.com/npp-plugins/plugintemplate/releases/download/v3.1/pluginTemplate.v3.1.bin.x64.zip or the outstanding update for n++ 7.7.